srcpkgs/xidlehook/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'xidlehook'
+pkgname=xidlehook
+version=0.10.0
+revision=1
+build_wrksrc="xidlehook-daemon"
+build_style=cargo
+configure_args="$(vopt_if audio '--features pulse' '--no-default-features')"
+hostmakedepends="pkg-config python3"
+makedepends="libxcb-devel libXScrnSaver-devel $(vopt_if audio pulseaudio-devel)"
+short_desc="Xautolock rewrite in Rust, with a few extra features"
+maintainer="MOIS3Y <stepan@zhukovsky.me>"
+license="MIT"
+homepage="https://gitlab.com/jD91mZM2/xidlehook"
+distfiles="${homepage}/-/archive/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
Could you remove the variables `homepage` and `pkgname` here, they are not variable.

Reply:
Done